ci-operator/config/rhpit/interop-tests/rhpit-interop-tests-main__simranpal-ocp4.13-lp-interop.yaml

```diff
@@ -0,0 +1,38 @@
+base_images:
+  cli:
+    name: "4.13"
+    namespace: ocp
+    tag: cli
+  os:
+    name: ubi
+    namespace: ocp
+    tag: "8"
+releases:
+  initial:
+    candidate:
+      product: ocp
+      stream: nightly
+      version: "4.13"
+  latest:
+    candidate:
+      product: ocp
+      stream: nightly
+      version: "4.13"
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: mtr-interop-aws
+  cron: 0 1 * * 1
+  steps:
+    cluster_profile: aws-cspi-qe
+    env:
+      BASE_DOMAIN: aws.interop.ccitredhat.com
```

Comment:
Please use the env var SPOT_INSTANCES: "true" when testing so that we save AWS costs.